cluster-autoscaler/simulator/predicatechecker/schedulerbased.go

```diff
@@ -53,15 +56,13 @@ func NewSchedulerBasedPredicateChecker(informerFactory informers.SharedInformerF
 		}
 	}
 
-	if len(schedConfig.Profiles) != 1 {
+	if len(schedConfig.Profiles) == 0 {
 		return nil, fmt.Errorf("unexpected scheduler config: expected one scheduler profile only (found %d profiles)", len(schedConfig.Profiles))
 	}
 	sharedLister := NewDelegatingSchedulerSharedLister()
 
-	framework, err := schedulerframeworkruntime.NewFramework(
-		context.TODO(),
-		scheduler_plugins.NewInTreeRegistry(),
-		&schedConfig.Profiles[0],
+	recorderFactory := func(string) events.EventRecorder { return nil }
+	profiles, err := scheduler_profile.NewMap(context.TODO(), schedConfig.Profiles, scheduler_plugins.NewInTreeRegistry(), recorderFactory,
```

Comment:
nit: can we make line breaking more consistent?

Reply:
ok